src/extensions/extension-store.ts

```diff
@@ -1,13 +1,8 @@
 import type { ExtensionId, ExtensionManifest, ExtensionVersion, LensExtension } from "./lens-extension";
```

Comment:
Do we still need this file?

Reply:
I left store related bits to this file but maybe we should remove it for now (until we really need it).